src/components/PlanningTable/Columns/Title.tsx

```diff
@@ -0,0 +1,38 @@
+import { type ColumnDef } from '@tanstack/react-table'
+import { TooltipProvider, Tooltip, TooltipTrigger, TooltipContent } from '@ttab/elephant-ui'
+import { cn } from '@ttab/elephant-ui/utils'
+import { type Planning } from '../data/schema'
+
+export const title: ColumnDef<Planning> = {
+  id: 'title',
+  accessorFn: (data) => data._source['document.title'][0],
+  cell: ({ row }) => {
+    const status = row.original._source['document.meta.core_planning_item.data.public'][0] === 'true'
+    const slugline = row.original._source['document.meta.core_assignment.meta.tt_slugline.value']
+
+    return (
+        <div className='flex space-x-2 w-fit'>
+          <TooltipProvider>
+            <Tooltip>
+              <TooltipTrigger>
+                <div className={cn(
```

Comment:
The code would be easier to read if we moved the `cn()` call outside of the JSX and store it as variable. Later on we should use `cva()` from `class-variance-authority` for similar class name conditionals.